src/Datadog.Trace.ClrProfiler.Managed/Integrations/Elasticsearch.Net/Pipeline.cs

```diff
@@ -59,7 +60,7 @@ public static object CallElasticsearchAsync<TResponse>(object pipeline, object r
 
             var cancellationToken = (cancellationTokenSource as CancellationTokenSource)?.Token ?? CancellationToken.None;
 
-            var originalMethod = DynamicMethodBuilder<Func<object, object, CancellationToken, TResponse>>.
+            var originalMethod = DynamicMethodBuilder<Func<object, object, CancellationToken, Task<TResponse>>>.
```

Comment:
Sorry, I don't have much context on this integration. But do we now have tests for integrations? And if yes why did this bug slip through?

Reply:
The tests were set up to only count the expected Spans, but ignored any exceptions. I fixed this in #194.